cmd/release/cmd/root.go

```diff
@@ -21,9 +28,9 @@ var rootCmd = &cobra.Command{
 // Execute adds all child commands to the root command and sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
+	cobra.OnInitialize(initConfig)
 	if err := rootCmd.Execute(); err != nil {
-		fmt.Println("error: " + err.Error())
-		os.Exit(1)
+		panic(err)
```

Comment:
Is the contents of the stack trace with a panic necessary here? Printing the error and exiting with a non 0 code is perferred.